yash.md

```diff
@@ -0,0 +1,82 @@
+# Yet Another SDR Handbook
+
+_A combination index, glossary, and high level guided tour for the Stanford Digital Repository_
+
+* Access Rights for digital content
```

Comment:
If there are hierarchical headings, I suggest we use formating to call them out.  Might I suggest bold?  And we could use italics for the answers.